PLAYER/SimulatePlayerInputGait.md

```diff
@@ -5,22 +5,27 @@ ns: PLAYER
 
 ```c
 // 0x477D5D63E63ECA5D 0x0D77CC34
-void SIMULATE_PLAYER_INPUT_GAIT(Player player, float amount, int gaitType, float speed, BOOL p4, BOOL p5);
+void SIMULATE_PLAYER_INPUT_GAIT(Player player, float amount, int gaitType, float rotationSpeed, BOOL p4, BOOL p5);
 ```
 
 ```
```

Comment:
Please don't make descriptions using code blocks.
Please also don't include examples in descriptions if you can add them as a proper example as shown [here](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md).